### PR TITLE
Modern theme

### PR DESCRIPTION
**Pull request by nojaf**
Update the default theme to use more modern web standards.
This cannot be merged until it is completed.
Alpha previews will be released from this branch.
This PR serves as a more convenient way to track the changes.

### CODE REVIEW

**Comment by nhirschey** [comment]
> * Replace `{{fsdocs-list-of-namespaces}}` with an empty string if no API docs are present.

:heart:

**Comment by nojaf** [comment]
Publish [v20.0.0-alpha-004](https://www.nuget.org/packages/fsdocs-tool/20.0.0-alpha-004)

> dotnet tool update fsdocs-tool --prerelease

## 20.0.0-alpha-004 - 2023-11-08

### Fixed
* Don't use font ligatures, the can confuse newcomers of F#.
* Replace `{{fsdocs-list-of-namespaces}}` with an empty string if no API docs are present.
* Improve default styling of `blockquote`
* Add some padding for level 3 and 4 headers in 'on this page' section.

File: docs/content/fsdocs-default.css [hunk 1]
```diff
@@ -119,26 +119,26 @@
     --link-color: #c5eff7;
     --search-background: #020202;
     --nav-category: rgb(207, 211, 215);
-    --code-strings-color: #5adfec;
+    --code-strings-color: #86b4b9;
     --code-printf-color: #6B2FBA;
```
Comment (smoothdeveloper) [inline docs/content/fsdocs-default.css:123]:
```suggestion
    --code-printf-color: #e786f4;
```

File: docs/content/fsdocs-default.css [hunk 2]
```diff
@@ -119,26 +119,26 @@
...
     --code-function-color: #6B2FBA;
-    --code-function2-color: #da9d9d;
+    --code-function2-color: #cbda9d;
     --code-activepattern-color: #4ec9b0;
     --code-unioncase-color: #4ec9b0;
     --code-enumeration-color: #8C6C41;
     --code-keywords-color: #a7c2f8;
-    --code-comment-color: #dcd8d8;
+    --code-comment-color: #84d16e;
     --code-operators-color: #b4c6ee;
```
Comment (smoothdeveloper) [inline docs/content/fsdocs-default.css:142]:
```suggestion
    --code-operators-color: #00d8ff;
```

File: docs/content/fsdocs-default.css [hunk 3]
```diff
@@ -119,26 +119,26 @@
...
     --code-numbers-color: #009999;
     --code-linenumbers-color: #80b0b0;
```
Comment (smoothdeveloper) [inline docs/content/fsdocs-default.css:144]: --code-mutable-color: ffb000;

**Comment by nojaf** [comment]
Hi @kMutagene and @nhirschey,

It might be more convenient if we merge this into the main branch.
I would like to have:
- https://github.com/fsprojects/FSharp.Formatting/pull/870
- https://github.com/fsprojects/FSharp.Formatting/pull/873
- https://github.com/fsprojects/FSharp.Formatting/pull/874
- https://github.com/fsprojects/FSharp.Formatting/pull/875

as part of v20, as they are great additions.

I would keep everything in alpha still, so we can revisit anything we want.
Sounds good?

**Comment by nojaf** [comment]
Alright, I'm calling it a day.
We can continue to work on this on the main branch if something still needs a tweak.

**Pull request merged**